backend/internal/interface/handler/update_stock_test.go

```
@@ -0,0 +1,94 @@
+		t.Run(c.name, func(t *testing.T) {
+			// ユースケースまで到達したら失敗と分かるよう、nil のまま呼び出す。
+			// 400 で弾かれていれば nil 参照は起きない。
+			h := &MedicationHandler{}
+			r := gin.New()
+			r.PATCH("/m/:medicationId/stock", func(ctx *gin.Context) {
+				ctx.Set("userID", "user-1")
+				h.UpdateStock(ctx)
+			})
+
+			w := httptest.NewRecorder()
+			req := httptest.NewRequest(http.MethodPatch, "/m/med-1/stock", strings.NewReader(c.body))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)